### DISCUSSION TITLE
Disable monitor autodetection?

### DISCUSSION THREAD

**Discussion by paholg** [hyprwm/Hyprland#4789] [Q&A] [answered]
Currently, when a monitor is added, hyprland detects it and automatically enables it. I would like to disable this behavior, any ideas how?

For context, my computer is attached to a monitor and a tv. I only ever want to use one of them at a time, with a key bind to switch. Currently, whenever the tv is turned on, it is enabled, even if I have previously run `hyprctl keyword monitor HDMI-A-1, disable`.

**Comment by vaxerski** [comment, marked as answer]
`monitor=,enabled,false`?

**Reply by paholg** [reply]
Would that mean I can still enable it with a `hyprctl keyword monitor` command? I don't want it permanently disabled, I just want to control precisely when it is enabled.

**Reply by vaxerski** [reply]
yes

**Reply by paholg** [reply]
The syntax seems to be `monitor=NAME,disabled`, but that works, thanks!
